quiz.css

```diff
@@ -5,19 +5,19 @@
     border-color: lightblue;
 }
 
-.div1{
+.section1{
 	border-style: dotted;
     border-color: lightgray;
 }
 
-.div2 {
+.section2 {
 	display:flex;
 	align-item:center;
```

Comment:
Did you mean `align-items: center`? Also, align is for vertical centering which I don't think you need here, simply the justify. Also, since this is a simple horizontal centering, you could try doing it without flexbox if you want.